### PR TITLE
[WIP] PR to fix the react testing library console warnings in the PR for issue-101

### PR DESCRIPTION
**Pull request by lpatmo**
Creating this PR so there's a git diff to easily refer back to.

### CODE REVIEW

**Comment by lpatmo** [comment]
Personal notes: https://github.com/facebook/react/issues/14769 is relevant

File: src/components/Auth/AuthForm.spec.js [hunk 1]
```diff
@@ -40,46 +40,48 @@ describe('AuthForm', () => {
 });
 
 describe('Signup', () => {
-  it('Register a new user on the signup form', async () => {
-    const { getByText, getByLabelText } = render(
-      <BrowserRouter>
-        <SignUpForm />
-      </BrowserRouter>
-    );
+  fit('Register a new user on the signup form', async () => {
```
Comment (sebbel) [inline src/components/Auth/AuthForm.spec.js:43]: fit? is this intended? :D

Comment (lpatmo) [reply src/components/Auth/AuthForm.spec.js:43]: Yeah, that was so I could test a single test at a time :) I pushed up a commit... one sec!

Comment (lpatmo) [reply src/components/Auth/AuthForm.spec.js]: I applied https://github.com/codebuddies/frontend/pull/111/commits/bf9c00b987bc3d9a50ab1b11ebdcd5654cc7cbba based on @gauravchl's suggestion in https://github.com/codebuddies/frontend/pull/105/commits/89b51566816ab1255d3376bd7bc0f36065cacc3b, but the interesting thing here is that if I run each test using `fit` individually, the tests pass without those warnings. 

But if I run `npm run test` with that commit in place, I still see: 
image

Not sure if there is something I'm obvious I missed, but tldr I'm very surprised to still see that error if I don't run each test individually...

Comment (lpatmo) [reply src/components/Auth/AuthForm.spec.js]: I recorded a quick screenrecording about the problem: https://gitduck.com/watch/5ea551ce1f9488f393e05e1a

**Comment by lpatmo** [comment]
Closing this out since Gaurav fixed these warnings in #105!

**Pull request closed without merging**